VERSION

```diff
@@ -1 +1 @@
-0.3.0
+0.4.0
```

Comment:
We shouldn't bump version in individual PRs. When we release (and how that impacts version) should be de-coupled from any one PR.